hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -18,3 +18,11 @@ Transition Checking->RomAhead "341->917"
 ANNOTATION: "UNR"
 Fsm state_d "1410404563"
 Transition KmacAhead->ReadingHigh "629->181"
+
+// In rom_ctrl_scrambled_rom, the wire scr_nonce is having a fixed value and is a part of RHS of 
+// the continuous assignment on line 81 and 82  
+CHECKSUM: "2714069379 225975262"
+ANNOTATION: "ModuleName: rom_ctrl_scrambled_rom"
```

Comment:
I think we can probably drop these two lines? (The annotation won't really do anything, and I don't think the checksum is something we need to track: it doesn't match the one earlier, which I suspect shows we should drop both :-D)